protocols/CartegeniaTree.sh

```diff
@@ -293,13 +283,12 @@ COUNTARRAY+=("Step 2.2.3(> 10 <20x AND Coding Effect); TRUE: ${count_2_2_3_true}
 
 ### 2.2.4 merge all variants that continue the tree
 #echo "${outputStep2_1}" "${outputStep2_2_1_overlap}" outputStep2_2_2_overlap outputStep2_2_3_overlap > "${outputFinalStep2}"
-java -cp ${EBROOTGATK}/${gatkJar} org.broadinstitute.gatk.tools.CatVariants \
-   -R ${indexFile} \
-   --variant "${outputStep2_1}" \
-   --variant "${outputStep2_2_1_nextHLA}" \
-   --variant "${outputStep2_2_3_nextHLA}" \
-   -out "${outputStep2_next}.tmp.vcf" \
-   --assumeSorted
+gatk MergeVcfs \
```

Comment:
according to this thread https://gatkforums.broadinstitute.org/gatk/discussion/12556/catvariants-mergevcfs-or-gathervcfs you should use GatherVcfs instead of MergeVcfs to replace the catvariants correctly

Reply:
AFAIK the GATK team is promoting ```GenomicsDB``` as the new way to go and that it should be the fastest. The guy in the thread complains it is slow though; maybe a hardware/config issue, so we should run some tests...